src/cheri/insns/addi4spn_16bit.adoc

```diff
@@ -31,7 +31,7 @@ Prerequisites::
 {c_cheri_default_ext_names}
 
 Included in::
-{rvyc_mod_xref}
+<<rvy_c_mod_insn_table>>
```

Comment:
this hasn't rendered correctly "section C.7" although it's not obvious to me why